### PR TITLE
Update and clean text inputs

### PR DESCRIPTION
**Pull request by matzimowski**
today:
screen shot 2016-07-01 at 12 52 44
yesterday:
screen shot 2016-07-01 at 12 54 38

### CODE REVIEW

File: src/components/form-elements/_text-input.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
-$inputHeight: rhythm(2);
-$inputLowHeight: rhythm(1.5);
```
Comment (aju) [inline src/components/form-elements/_text-input.scss]: No more low / normal ?

File: src/components/form-elements/_text-input.scss [hunk 2]
```diff
@@ -66,48 +59,27 @@ $includeHtml: false !default;
...
+        padding: 0 gutter(1);
 
         &::placeholder {
           font-size: fontSize(standout);
         }
       }
     }
 
-    &--subtle-border {
-      $boxShadowColor: rgba(0, 0, 0, 0.2);
```
Comment (aju) [inline src/components/form-elements/_text-input.scss]: ??? why?

File: src/components/form-elements/_text-input.scss [hunk 3]
```diff
@@ -66,48 +59,27 @@ $includeHtml: false !default;
...
-      border: none;
-      box-shadow: 0 0 10px $boxShadowColor;
-      transition: box-shadow 300ms;
-
-      &:focus {
-        box-shadow: 0 0 10px $boxShadowColorFocus;
-      }
-    }
-
-    &--transparent-border {
```
Comment (aju) [inline src/components/form-elements/_text-input.scss]: why ??

File: src/components/form-elements/_text-input.scss (hunk 1 above)
Comment (matzimowski) [inline src/components/form-elements/_text-input.scss]: nope. normal has the same heights as low before.

File: src/components/form-elements/_text-input.scss (hunk 2 above)
Comment (matzimowski) [inline src/components/form-elements/_text-input.scss]: we do not need this modificators anymore.

File: src/components/form-elements/_text-input.scss (hunk 3 above)
Comment (matzimowski) [inline src/components/form-elements/_text-input.scss]: we do not need this modificators anymore.

**Comment by aju** [comment]
Ubercool

**Comment by vergilius** [comment]
**F** antastic
**A** mazing
**N** everending
**C** coding
**Y OLO**

💛  👍

**Pull request merged**